### ISSUE TITLE
Doesn't block video paused continue watching popup

### ISSUE THREAD

**Issue by speedycatfish** [Araxeus/Youtube-Volume-Scroll#11]
Title. I'll leave my music playlist on while playing a game, and the popup will stop my playlist. I am on Windows 10 using the latest version of Edge.

**Comment by Araxeus** [comment]
So you are saying it get paused after more than 15min? that's shouldn't happen...

could it be that you have another tab selected? 
does this happen all the time or only happened once?

**Comment by speedycatfish** [comment]
Yeah, sometimes I have another tab selected. Does this extension only block the popup if youtube is the tab open? And yes, it has happened several times.
